extras/common/app/models/search_builder.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
```

Comment:
Adding this to resolve error in my local env on /advanced for Cinefiles and PAHMA. BAMPFA has its own version of search_builder.rb.